setup.cfg

```diff
@@ -22,8 +22,7 @@ packages =
     lavalink
 python_requires = >=3.5.3
 install_requires =
-    websockets>=8.1
-    discord.py>=1.2.5
+    discord.py>=1.4.0
```

Comment:
Even though discord.py requires aiohttp as a dependency, it's always best to explicitly specify all dependencies you use directly, e.g. aiohttp in this case. Thanks for adding discord.py however.

Reply:
Fixed. 😄

Reply:
Preda given that RLL has been toggled to be used strictly by Red, can you please bump the python Version to Match Reds?  I.e >=3.8.1, and not a point of action, but the install requirements here probably could just be straight up removed, and let Red handle them.

Reply:
Actually, don't I'll do a follow up PR, at it will require more than just a single line change here due to the test. But we already agreed to make the requirements to be handled by Red, and bump python to 3.8.1 to keep it in line with Red, just had some issues with the test so it was postponed and forgotten about